js/main.js

```diff
@@ -3,20 +3,53 @@
 {
   const startBtn = document.getElementById('start');
   const caption = document.getElementById('caption');
+  const content = document.getElementById('content');
+  const ganre = document.getElementById('ganre');
 
+  
+  class Quiz {
+    constructor(quizData) {
+      this._quizz = quizData;
+      this._correctAnswersNum = 0;
+    }
+    getQuizCategory(index) {
+      return this._quizz[index - 1].category;
+    }
+    getQuizQuestion(index) {
+      return this._quizz[index - 1].question;
+    }
+    getQuizDifficulty(index) {
+      return this._quizz[index - 1].difficulty;
+    }
+    getQuizCorrect_answer(index) {
+      return this._quizz[index - 1].correct_answer;
+    }
+    getQuizIncorrect_answers(index) {
+      return this._quizz[index - 1].incorrect_answers;
+    }
```

Comment:
>今回の変更で クイズの JSON が一つだけ 入るように修正したわけですので
>添え字でアクセスしているところ（ this._quizzes[index - 1] というコード）は全て書き換える必要があります 

*変数の名前* を書き換えられたようですが、対応いただきたいのはそこではなくて
以下のようにして「クイズのオブジェクト配列にアクセス」するコードではなくて
「クイズオブジェクトにアクセス」するコードに書き換えてほしいという意味でした。

```suggestion
    getQuizCategory(index) {
      return this._quizz.category;
    }
    getQuizQuestion(index) {
      return this._quizz.question;
    }
    getQuizDifficulty(index) {
      return this._quizz.difficulty;
    }
    getQuizCorrect_answer(index) {
      return this._quizz.correct_answer;
    }
    getQuizIncorrect_answers(index) {
      return this._quizz.incorrect_answers;
    }
```